### PR TITLE
[diem framework] Bulk script fun conversion 

### PR DESCRIPTION
**Pull request by tzakian**
This PR performs the conversion from transaction scripts over to script functions in modules in the Diem Framework.

The build system and documentation generation for the Diem Framework have been updated to support this as well in this PR.

### CODE REVIEW

**Comment by libra-action** [comment]
**This PR may have modified JSON-RPC server code.**

Breaking changes policy:

1. Chaning JSON-RPC method signatures or removing/modifying fields
   in the response in /V1 will not be accepted.
2. Adding fields is ok.
3. Adding a new JSON-RPC method is ok.

Please ensure you have also done the following to update the docs:

1. Update json-rpc/API-CHANGELOG.md.
2. Add/update type/method documentation under json-rpc/docs.

File: json-rpc/API-CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,16 @@ Please add the API change in the following format:
 
 ```
 
+## 2021-03-16 Add support for script functions in `TransactionDataView`
+
+The `TransactionDataView` API remains unchanged, but it now supports user
+transactions using the new script function transaction payload. The script
+names are the fully qualified name of the Move function being called by the
+script function, i.e., of the form
+`<short_lossless_address>::<move_module_name>::<move_function_in_module`. The
```
Comment (sblackshear) [inline json-rpc/API-CHANGELOG.md:22]:
```suggestion
`<short_lossless_address>::<move_module_name>::<move_function_in_module>`. The
```

File: json-rpc/tests/integration_test.rs [hunk 2]
```diff
@@ -569,6 +569,14 @@ fn create_test_cases() -> Vec<Test> {
                 });
             },
         },
+        Test {
+            name: "Upgrade diem version",
+            run: |env: &mut testing::Env| {
+                let script = tmp_new_transaction_script_builders::encode_update_diem_version_script(0, 2);
```
Comment (sblackshear) [inline json-rpc/tests/integration_test.rs:575]: Will this always run in a state where the current Diem version is 1?

File: language/diem-framework/modules/AccountAdministrationScripts.move [hunk 3]
```diff
@@ -0,0 +1,534 @@
+address 0x1 {
+/// This module holds transactions that can be used to administer accounts in the Diem Framework.
```
Comment (sblackshear) [inline language/diem-framework/modules/AccountAdministrationScripts.move:2]: Might help to add one or two examples of what administration means (e.g., key rotation).

**Review by sblackshear — APPROVED** [review]

**Pull request closed without merging**